writer/marshal_test.go

```diff
@@ -0,0 +1,35 @@
+package writer
+
+import (
+	"testing"
+)
+
+type marsh struct{}
+
+func (m marsh) MarshalStarlark() ([]byte, error) {
```

Comment:
`func (marsh) MarshalStarlark() ([]byte, error) …`

(it doesn't need the receiver)